netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -322,6 +322,16 @@ public ConnectionClientTransport newClientTransport(
       }
       ProtocolNegotiator negotiator = protocolNegotiator != null ? protocolNegotiator :
           createProtocolNegotiator(authority, negotiationType, sslContext);
+      String proxy = System.getenv("GRPC_PROXY_EXP");
```

Comment:
Experimental. I agree it isn't clear, but it's also going to be trashed. There's another PR that's currently out that uses the exp abbreviation for ALPN.